Debug App/Podfile.lock

```diff
@@ -38,4 +38,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d4a25e6279c7edfc36cd259108d9afa9f81e1da5
 
-COCOAPODS: 1.15.0
+COCOAPODS: 1.13.0
```

Comment:
This happens because someone is running `pod install` without `bundle exec`

[Our locked version is 1.13.0](https://github.com/primer-io/primer-sdk-ios/blame/master/Gemfile.lock#L36)

We can discuss and change this but I'd rather not do it here.